docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -91,42 +93,36 @@ In our example, we will be using `java-spring-boot` to deploy a sample https://s
 
 == Deploying a Java Spring Boot® component to an OpenShift cluster
 
-In this example we will be deploying an https://github.com/odo-devfiles/springboot-ex[example Spring Boot® component] that uses https://maven.apache.org/install.html[Maven] and Java 8 JDK.
+In this example we will be deploying a sample project associated with the java-spring-boot devfile that uses https://maven.apache.org/install.html[Maven] and Java 8 JDK.
 
-. Download the example Spring Boot® component
+. Create a directory to store the source code for the component:
 +
 [source,sh]
 ----
- $ git clone https://github.com/odo-devfiles/springboot-ex
+ $ mkdir <directory-name>
 ----
 
-. Change the current directory to the component directory:
+. Create a component configuration of Spring Boot component-type named myspring and download its sample project:
 +
 [source,sh]
 ----
- $ cd <directory-name>
-----
+   $ odo create java-spring-boot myspring --downloadSource
+   Experimental mode is enabled, use at your own risk
 
-. List the contents of the directory to confirm that the front end is indeed a Java application:
-+
-[source,sh]
-----
- $ ls
- chart  Dockerfile  Dockerfile-build  Dockerfile-tools  Jenkinsfile  pom.xml  README.md  src
+   Validation
+    ✓  Checking devfile compatibility [195728ns]
+    ✓  Creating a devfile component from registry: DefaultDevfileRegistry [170275ns]
+    ✓  Validating devfile component [281940ns]
+
+    Please use `odo push` command to create the component with source deployed
 ----
 
-. Create a component configuration using the `java-spring-boot` component-type named `myspring`:
+. List the contents of the directory to see that the devfile and sample Java application was downloaded:
```

Comment:
:+1: I like this part despite my previous comment above ^^. But I think we could perhaps include both options. Saying that you can either `git clone` or use `--downloadSource`.

Reply:
How about I revert to using `git clone`, but add a little blurb about the flag?